### PR TITLE
CONCF-607 Show items from all namespaces in Curated Content

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/CONCF-607

/cc @bognix @vforge @tomasznapieralski

### CODE REVIEW

File: front/scripts/main/components/CuratedContentItemComponent.ts [hunk 1]
```diff
@@ -21,6 +21,24 @@ App.CuratedContentItemComponent = Em.Component.extend(App.ViewportMixin, {
 	href: Em.computed.oneWay('model.url'),
 	isArticle: Em.computed.equal('model.type', 'article'),
 
+	icon: Em.computed('type', function (): string {
```
Comment (hakubo) [inline front/scripts/main/components/CuratedContentItemComponent.ts]: perfect function to write a unit test :)

Comment (rogatty) [inline front/scripts/main/components/CuratedContentItemComponent.ts]: Added!

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**